mappings/net/minecraft/util/math/MutableIntBoundingBox.mapping

```diff
@@ -57,3 +57,8 @@ CLASS net/minecraft/class_3341 net/minecraft/util/math/MutableIntBoundingBox
 		ARG 2 minZ
 		ARG 3 maxX
 		ARG 4 maxZ
+	METHOD method_19311 withOffset (III)Lnet/minecraft/class_3341;
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+	METHOD method_19635 getCenter ()Lnet/minecraft/class_2382;
```

Comment:
Since this returns an integer 3-d vector, what does this return when the the box is from (0, 0, 0) to (1, 1, 1)?

Reply:
(0, 0, 0). It's rounded down as usual.